values.yaml

```diff
@@ -133,7 +133,7 @@ server:
     enabled: false
     labels: {}
       # traffic: external
-    annotations: {}
+    annotations: []
```

Comment:
I think we'll want to keep annotations defaulting to `{}` as described in #272.

Reply:
I agree that using {} is better and is standard practice in most charts, and it works correctly when using toYaml on the annotations, but as this current chart expects the annotations as a multi-line string it gave the warning. The PR you mentioned came in after mine, so personally yes I would keep the {} now to allow toYaml to work and just put up with the warning.